Dockerfile

```diff
@@ -11,8 +11,10 @@ FROM python:${PYTHON_VERSION} as run
 
 WORKDIR /app
 
+
```

Comment:
Consider adding a `.dockerignore` file to exclude unnecessary files from being copied into the Docker image. This can help reduce the image size and improve build performance.